src/api/config.json

```diff
@@ -1,16 +1,16 @@
 {
```

Comment:
We can actually provide more than one sample config.json, corresponding to few popular brokers. My simple reasoning - many people like instant gratification :)